### PR TITLE
User Engagement in Service Delivery blog

### PR DESCRIPTION
**Pull request by ElminaIusifova**

### CODE REVIEW

**Comment by ElminaIusifova** [comment]
@gabrielcossette @gcharest  the length of the card is too long on the webpage for this blog, can one of you help me to fix it please? Thanks

**Comment by gcharest** [comment]
> @gabrielcossette @gcharest the length of the card is too long on the webpage for this blog, can one of you help me to fix it please? Thanks

Oh, yes! Absolutely, I think it's due to the fact we haven't identified any "excerpt" part. What would you like to display on the card? The first two sentences only?

**Review by remyb2canada — APPROVED** [review]
Great work Elmina!

**Comment by ElminaIusifova** [comment]
@gcharest Yes please, first 2-3 sentences as long as the card with this blog is the same length as others. Thank you

**Review by gabrielcossette — APPROVED** [review]
Sorry for the delay.
Great blog post! I reviewed and made some adjustments. Feel free to change them back or else.

**Review by gcharest — CHANGES_REQUESTED** [review]
Making a few changes to adjust the blog card!

**Review by gcharest — CHANGES_REQUESTED** [review]
All good, tests will pass once we merge #1860.

- [ ] Need French translation! :smile:

**Comment by gabrielcossette** [comment]
> 
> 
> Making a few changes to adjust the blog card!

Thanks for fixing this Guillaume, I forgot about it! 🙂

**Review by gcharest — APPROVED** [review]

File: _posts/en/2021-08-18-user-engagement-in-service-delivery.md [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+The main user of the tool is a public servant in IITB looking to continuously improve and support the efforts of its peers.
+By understanding the expectations, concerns, and motivations of our target users, we came up with a design-focused solution.
+To design for a public servant while being public servants ourselves, it is so easy to create a self-referential product as if we are making the product only for us, when in fact the target audience may be quite unlike us.
+We also created a mock-up which was recently presented to a real group of users in order to collect feedback for further improvement.
+
+Thinking about the needs of our fictional – end user persona, we tried to shape the product strategy and create the best mock-up for *him*.
+It is worth mentioning that although user personas helped us to prioritize the features, it was not the only tool used for prioritization.
+We were constantly collecting feedback from stakeholders to keep a proper balance between the needs of the business and the needs of the users, in order to create a solution that would satisfy both.
+With the feedback received, we tried to iterate the overall experience for the user.
```
Comment (gabrielcossette) [inline _posts/en/2021-08-18-user-engagement-in-service-delivery.md:62]: I forgot to leave a comment on this. Not sure "iterate" is the right verb here, but I understand the idea behind it. Maybe it needs rephrasing.

File: _posts/en/2021-09-16-user-engagement-in-service-delivery.md [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+We created personas to help us get to know our customers, their behaviour, goals, frustrations, to understand who we are designing for, and if it meets the needs of our target audience.
+
+This is what happened during the [DevOps Self-Assessment tool](https://sara-sabr.github.io/ITStrategy/devops-self-assessment.html) design sprint.
+Based on research, we created fictional characters to help us understand the users we are designing for.
+We discovered different types of users with their behaviour patterns, goals, skills, attitudes, and background information, as well as the environment in which they operate to see where we should focus and improve while designing our application.
+The main user of the tool is a public servant in IITB looking to continuously improve and support the efforts of its peers.
+By understanding the expectations, concerns, and motivations of our target users, we came up with a design-focused solution.
+To design for a public servant while being public servants ourselves, it is so easy to create a self-referential product as if we are making the product only for us, when in fact the target audience may be quite unlike us.
```
Comment (remyb2canada) [inline _posts/en/2021-09-16-user-engagement-in-service-delivery.md:56]: I REALLY REALLY LIKE this section.
Nice

Comment (remyb2canada) [inline _posts/en/2021-09-16-user-engagement-in-service-delivery.md:56]: I REALLY REALLY like this sentence.
Nice.

**Review by remyb2canada — APPROVED** [review]

**Comment by gabrielcossette** [comment]
FR translation review completed 🙂

**Pull request merged**